<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fenetre</class>
 <widget class="QWidget" name="Fenetre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fenetre</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>431</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayoutLibelleMaqueDescription">
    <item row="0" column="0">
     <widget class="QLabel" name="labelLibelle">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Libellé du produit	&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelDescription">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Description		&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditMarque"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelMarque">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Marque	&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditLibelle"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditDescription"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>410</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayoutPrix">
    <item row="0" column="0">
     <widget class="QLabel" name="labelPrix">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Prix&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditPrix"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonDeleteFromDB">
   <property name="geometry">
    <rect>
     <x>286</x>
     <y>530</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Supprimer de la BDD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonInsertToDB">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>530</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Ajouter à la BDD</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Gestion de produits - Supermarché&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClearInputs">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>500</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Supprimer les champs</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelJeVeux">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Je veux :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxFeature">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>311</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ajouter un produit à la base de données</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Supprimer un produit de la base de données</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Consulter la base de données</string>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="tableWidgetDBTable">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>380</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="tabKeyNavigation">
    <bool>false</bool>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxUpdateDB">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15px;
color:black;</string>
   </property>
   <property name="text">
    <string>Modifier la BDD</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWhoAmI">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>560</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>tableWidgetDBTable</zorder>
  <zorder>gridLayoutWidget</zorder>
  <zorder>gridLayoutWidget_2</zorder>
  <zorder>pushButtonDeleteFromDB</zorder>
  <zorder>pushButtonInsertToDB</zorder>
  <zorder>labelTitle</zorder>
  <zorder>pushButtonClearInputs</zorder>
  <zorder>labelJeVeux</zorder>
  <zorder>comboBoxFeature</zorder>
  <zorder>checkBoxUpdateDB</zorder>
  <zorder>labelWhoAmI</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
